docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2220,6 +2220,10 @@ For more information about Grafana Enterprise, refer to [Grafana Enterprise]({{<
 
 Keys of alpha features to enable, separated by space.
 
+### FEATURE_TOGGLE_NAME = false 
+  
+Some feature toggles for stable features are on by default. Use this setting to explicitly disable an on-by-default feature toggle with the name FEATURE_TOGGLE_NAME, for example `exploreMixedDatasource = false`.
```

Comment:
we may also make this a but more explicit, like this maybe (but its pretty inconsistent compared to the rest of the page)

![image](https://github.com/grafana/grafana/assets/1170767/826e6861-b8a9-4e8c-8b56-00e479e4adac)

Reply:
there is similar language in:
https://github.com/grafana/grafana/blob/main/conf/defaults.ini#L1492

that gets included in the depolyments